index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Setup Mobile Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <article>
+        <header>   
+            <time class="time"><b>9:41</b></time>
+        <span>
+            <img class="menu-icon" src="/images/Icon - Menu.svg" alt="hamburger list">
+            <img class="pwr" src="/images/Battery.svg" alt="battery">
+            <img class="wifi" src="/images/Wifi.svg" alt="wifi">
+            <img class="cell-connection" src="/images/Cellular Connection.svg" alt="cell connection">
+        </span>
+            <p class="logo">My Logo</p>  
+            <h2>I'm Mariane Glad to see you!</h2>
+            <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
```

Comment:
- To improve your site's HTML structure, I advise that you remove these lines and include them in the section below. The `header` tag practically should contain the site's navigation. You can refer to this [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info 👍 👍 
- Also, for semantic reasons, I suggest using an `h1` in place of the `h2` tag, You can refer to this [article](https://www.reliablesoft.net/h1-tag/)
- There are little differences in design between your site and the Figma design, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Figma design:
- Please reference the marked section in the Figma design below
![rjrjrj](https://user-images.githubusercontent.com/73607512/205679908-023835dd-f1a6-400d-a2b0-480d8407292b.PNG)
Your design:
![hthth](https://user-images.githubusercontent.com/73607512/205679989-855476da-bd62-4f45-910d-562079004a56.PNG)

Reply:
In applying your final recommendation regarding the border-radius on the left side corner (highlighted in blue), I would have to export the Figma image that includes elements that are not required for the project (Highlighted in yellow). I kindly plea you advise further. 
![Screenshot_2](https://user-images.githubusercontent.com/110445380/206103301-ef7502b4-db74-4d79-a0e3-4889aa864610.png)